api/metrics-api-jaxrs/src/main/java/org/hawkular/metrics/api/jaxrs/handler/MetricHandler.java

```diff
@@ -70,33 +71,32 @@
     @GET
     @Path("/")
     @ApiOperation(value = "Find tenant's metric definitions.", notes = "Does not include any metric values. ",
-            response = List.class, responseContainer = "List")
-    @ApiResponses(value = { @ApiResponse(code = 200, message = "Successfully retrieved at least one metric "
-            + "definition."),
-            @ApiResponse(code = 204, message = "No metrics found."),
-            @ApiResponse(code = 400, message = "Given type is not a valid type.", response = ApiError.class),
+                  response = List.class, responseContainer = "List")
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Successfully retrieved at least one metric definition.",
+                         response = ApiError.class),
+            @ApiResponse(code = 204, message = "No metrics found.",
+                         response = ApiError.class),
```

Comment:
Le 22/06/2015 19:21, jsanda a écrit :
> Do you mean that there should be no response body?

Yes. Have a look at `ApiUtils#noContent`.

Actually, it behaves like this since the beginning. I think it's the 
right thing to do, it's not an error, it's absence of result.